clients/src/test/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThreadTest.java

```diff
@@ -97,7 +97,7 @@ public void testStartupAndTearDown() throws InterruptedException {
         assertFalse(backgroundThread.isRunning());
 
         // There's a nonzero amount of time between starting the thread and having it
-        // begin to execute our code. Wait for a bit before checking...
+        // begins to execute our code. Wait for a bit before checking...
```

Comment:
I assert "begin" is correct as is here.

Reply:
🤦 - it was an auto correction thing from one of the plugin i have.